CHANGELOG.rst

```diff
@@ -9,26 +9,64 @@ Release 2.2.0
 Library Features
 ^^^^^^^^^^^^^^^^
 
+- Indices:
+    - FM index now has several options to reduce space consumption or improve running time
+        - up to three level rank dictionaries
+        - size of blocks on the lowest level (referred to as ``WORDS_PER_BLOCK``)
+    - Bidirectional FM index with constant running time using EPR-dictionaries
+    - Please see the `manual <seqan.readthedocs.io/en/master/Tutorial/DataStructures/Indices/FMIndex.html>`_ for more information
+
 - Alignment:
     - Vectorized DP-Alignment algorithms using SSE3/AVX2. Allows for inter-parallel alignment computation in a many-vs-many or one-vs-many mode.
+    - add a scoring matrix type that can be specified at runtime (e.g. BLOSSUM62, BLOSSUM50)
 
 - Modifier:
     - ModifiedString ModPadding: Expand a string with padding symbols, without changing the source.
 
+- Other:
+    - Replace p_thread implementation with STLs thread support library. Increases performance and fixes rare bugs.
```

Comment:
- Replace pthread implementation with STLs thread support library. Increases performance and fixes rare bugs in **bam_io**.